README.md

```diff
@@ -150,3 +150,30 @@ When running the app locally, ensure you are using the development `GOOGLE_SHEET
 (See google drive for this)
 
 If updating your environment variable doesn't work, try `rm -rf _build` and then starting your server again
+
+### Image uploads
+
+##### File size
+
+The Wagtail cms allows for a maximum file size of 10MB.
+
+Althouth I would suggest that files be no bigger than 1MB.
+
+Be aware though that many uploads of high resolution images will result in increased costs from our file hosting provider.
```

Comment:
More importantly, users on poor connections or on mobile data will experience slow performance and increased data download costs.